data/tabular/bioavailability_ma_et_al/transform.py

```diff
@@ -119,22 +119,76 @@ def get_and_transform_data():
 journal = {Journal of Pharmaceutical and Biomedical Analysis}""",
         ],
         "templates": [
-            "The molecule with the {SMILES__description} representation {SMILES#} has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
-            "Based on the {SMILES__description} representation {SMILES#}, the molecule has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
-            "The {SMILES__description} {SMILES#} represents a molecule that has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
+            "The molecule with the {SMILES__description} {#representation of |!}{SMILES#} has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
+            "Based on the {SMILES__description} {#representation of |!}{SMILES#}, the molecule has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
+            "The {SMILES__description} {SMILES#} {#represents|is from!} a molecule that has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
```

Comment:
```SMILES is from``` a molecule doesn't sound too good to me. I propose:

```suggestion
            "The {SMILES__description} {SMILES#} {#represents|is!} a molecule that has a {bioavailable#low&high} {bioavailable__names__noun}.",  # noqa: E501
```